packages/app/obojobo-express/express_current_user.js

```diff
@@ -5,6 +5,7 @@ const logger = oboRequire('logger')
 const setCurrentUser = (req, user) => {
 	if (!(user instanceof User)) throw new Error('Invalid User for Current user')
 	req.session.currentUserId = user.id
```

Comment:
Do we need to set this? It seems to only be used in this file.

Reply:
req.session.currentUserId?  yea, this is where the server links the current session to the user (req.session is written to the db)